ui_framework/doc_site/src/components/guide_section/guide_section_container.js

```diff
@@ -0,0 +1,25 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { GuideSection } from './guide_section.jsx';
+
+import {
+  openCodeViewer,
+  registerSection,
+  unregisterSection,
+} from '../../actions';
+
+function mapStateToProps(state, ownProps) {
+  return ownProps;
+}
```

Comment:
For the simple cases `mapDispatchToProps` can just be an object that lists the actions (it's just complex cases that need the functions and use of `bindActionCreators`). This is how we would write this file in Cloud UI:

```js
import { connect } from 'react-redux';

import { GuideSection } from './guide_section.jsx';
import {
  openCodeViewer,
  registerSection,
  unregisterSection,
} from '../../actions';

// does not need any state, receives everything through props
const mapStateToProps = null

export default connect(
  mapStateToProps,
  {
    openCodeViewer,
    registerSection,
    unregisterSection,
  }
)(GuideSection);
```

(without the comment, but that was just to make it clearer)